doc/tutorial/02_layout/index.md

```diff
@@ -0,0 +1,298 @@
+## X-HIBF = An HIBF with tmax = X, e.g a maximum of X technical bins on each level.
+## ### Column Description ###
+## tmax : The maximum number of technical bin on each level
+## c_tmax : The technical extra cost of querying an tmax-IBF, compared to 64-IBF
+## l_tmax : The estimated query cost for an tmax-HIBF, compared to an 64-HIBF
+## m_tmax : The estimated memory consumption for an tmax-HIBF, compared to an 64-HIBF
+## (l*m)_tmax : Computed by l_tmax * m_tmax
+## size : The expected total size of an tmax-HIBF
+# tmax	c_tmax	l_tmax	m_tmax	(l*m)_tmax	size
+64	1.00	2.00	1.00	2.00	12.8MiB
+# Best t_max (regarding expected query runtime): 64
+```
+And afterwards:
+```bash
+raptor layout --input-file all_bin_paths.txt --tmax 64
+```
+Your directory should look like this:
+```bash
+$ ls
+1024/                    all_bin_paths.txt        chopper_sketch.count     mini/
+64/                      binning.out              chopper_sketch_sketches/
+```
+
+\note
+We will use this mini-example in the following, both with further parameters and then for `raptor index --hibf`.
+Therefore, we recommend not deleting the files including the built indexes.
+
+\endsolution
+
+## Additional parameters
+
+To create an index and thus a layout, the individual samples of the data set are chopped up into k-mers and determine in
+their so-called bin the specific bit setting of the Bloom Filter by passing them through hash functions. This means that
+a k-mer from sample `i` marks in bin `i` with `j` hash functions `j` bits with a `1`.
+If a query is then searched, its k-mers are thrown into the hash functions and looked at in which bins it only points
+to ones. This can also result in false positives. Thus, the result only indicates that the query is probably part of a
+sample.
+
+This principle also applies to the Hierarchical Interleaved Bloom Filter, except that the bins are then stored even more
+efficiently as described above and this is described by the layout. This means that you already have to know some
+parameters for the layout, which you would otherwise specify in the index:
+
+With `--kmer-size` you can specify the length of the k-mers, which should be long enough to avoid random hits.
+By using multiple hash functions, you can sometimes further reduce the possibility of false positives
+(`--num-hash-functions`). We found a useful [Bloom Filter Calculator](https://hur.st/bloomfilter/) to get a calculation
+if it could help. As it is not ours, we do not guarantee its accuracy.
+
+Each Bloom Filter has a bit vector length that, across all Bloom Filters, gives the size of the Interleaved Bloom
+Filter, which we can specify in the IBF case. Since the HIBF calculates the size of the index itself, it is no longer
+possible to specify a size here. But we can offer the option to name the desired false positive rate with
+`--false-positive-rate`.
+
+\note These parameters must be set identically for `raptor index`.
+
+\todo This is not checked at the moment?
```

Comment:
Not yet, the parameters from `build`/`layout` will be stored in the index once we move away from `--hibf` being advanced